src/frontend/components/Dashboard/index.tsx

```diff
@@ -19,24 +19,27 @@ const DashboardContainer = styled.div`
 
 /** Props shape for the Dashboard component. */
 interface DashboardProps {
-  video?: Video;
-  document?: Document;
-  objectType: modelName;
+  object: Document | Video;
+  objectType?: modelName;
+  children?: React.ReactNode;
 }
 
 /** Component. Displays a Dashboard with the state of the video in marsha's pipeline and provides links to
  * the player and to the form to replace the video with another one.
  * Will also be used to manage related tracks such as timed text when they are available.
  * @param video The video object from AppData. We need it to populate the component before polling starts.
  */
-const Dashboard = ({ document, video, objectType }: DashboardProps) => {
+const Dashboard = ({ object, objectType, children }: DashboardProps) => {
```

Comment:
I think now the Dashboard should be only this:

```
interface DashboardProps {
  object: Document | Video;
  children?: React.ReactNode;
}

const Dashboard = ({ object, children }: DashboardProps) => {
  return (
    <DashboardContainer>
      <LTINav object={object} />
      {children}
    </DashboardContainer>
  );
};
```

And in the LTIRoutes, for the dashboard route matching part you render it like this

```
<Route
          exact
          path={DASHBOARD_ROUTE()}
          render={() => {
            if (appData.modelName === modelName.DOCUMENTS) {
              return (
                <Dashboard object={appData.document!}>
                  <DashboardDocument document={appData.document!} />
                </Dashboard>
              );
            }

            if (appData.modelName === modelName.VIDEOS) {
              return (
                <Dashboard object={appData.video!} >
                  <DashboardVideo video={appData.video!} />
                </Dashboard>
              );
            }

            return <Redirect push to={FULL_SCREEN_ERROR_ROUTE('notFound')} />;
          }}
        />
```